bluesky_queueserver/manager/worker.py

```diff
@@ -412,10 +401,7 @@ def _command_pause_plan_handler(self, *, option):
                 err_msg = str(ex)
         else:
             status = "rejected"
-            err_msg = (
-                "Run engine can be paused only in 'running' state. "
-                f"Current state: '{self._RE._state}'"
-            )
+            err_msg = "Run engine can be paused only in 'running' state. " f"Current state: '{self._RE._state}'"
```

Comment:
```suggestion
            err_msg = "Run engine can be paused only in 'running' state. Current state: '{self._RE._state}'"
```

Reply:
This change drops the f

Reply:
Oh, right. I forgot to move it to the very beginning.